ethnodes/ethnodes-entrypoint.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
```

Comment:
too much going on here for me to process right now. I'll try to take another look tomorrow.

Reply:
This file is based on the official mysql container build script. I added the source and changes I made at the top of this file.